algoliasearch/type_query.go

```diff
@@ -0,0 +1,60 @@
+package algoliasearch
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+type MultipleQueriesRes struct {
+	Results []MultipleQueryRes `json:"results"`
```

Comment:
From what I remember we had this discussion already and you said that staying as close to the API as possible was better, this is why I did that. However I quite agree that this structure is useless.